crates/qt-build-utils/src/lib.rs

```diff
@@ -1059,95 +519,30 @@ Q_IMPORT_PLUGIN({plugin_class_name});
         }
     }
 
+    /// Create a [QtToolRcc] for this [QtBuild]
+    ///
+    /// This allows for using [rcc](https://doc.qt.io/qt-6/resources.html)
+    pub fn rcc(&self) -> QtToolRcc {
+        QtToolRcc::new(self.qt_installation.as_ref())
+    }
+
     /// Run [rcc](https://doc.qt.io/qt-6/resources.html) on a .qrc file and save the output into [cargo's OUT_DIR](https://doc.rust-lang.org/cargo/reference/environment-variables.html).
     /// The path to the generated C++ file is returned, which can then be passed to [cc::Build::files](https://docs.rs/cc/latest/cc/struct.Build.html#method.file).
     /// This function also returns a String that contains the name of the resource initializer
     /// function.
     /// The build system must ensure that if the .cpp file is built into a static library, either
     /// the `+whole-archive` flag is used, or the initializer function is called by the
     /// application.
-    pub fn qrc(&mut self, input_file: &impl AsRef<Path>) -> Initializer {
-        if self.rcc_executable.is_none() {
-            self.rcc_executable = Some(self.get_qt_tool("rcc").expect("Could not find rcc"));
-        }
-
-        let input_path = input_file.as_ref();
-        let output_folder = PathBuf::from(&format!(
-            "{}/qt-build-utils/qrc",
-            env::var("OUT_DIR").unwrap()
-        ));
-        std::fs::create_dir_all(&output_folder).expect("Could not create qrc dir");
-        let output_path = output_folder.join(format!(
-            "{}.cpp",
-            input_path.file_name().unwrap().to_string_lossy(),
-        ));
-        let name = input_path
-            .file_name()
-            .unwrap()
-            .to_string_lossy()
-            .replace('.', "_");
-
-        let cmd = Command::new(self.rcc_executable.as_ref().unwrap())
-            .args([
-                input_path.to_str().unwrap(),
-                "-o",
-                output_path.to_str().unwrap(),
-                "--name",
-                &name,
-            ])
-            .output()
-            .unwrap_or_else(|_| panic!("rcc failed for {}", input_path.display()));
-
-        if !cmd.status.success() {
-            panic!(
-                "rcc failed for {}:\n{}",
-                input_path.display(),
-                String::from_utf8_lossy(&cmd.stderr)
-            );
-        }
-
-        let qt_6_5 = SemVer {
-            major: 6,
-            minor: 5,
-            ..SemVer::default()
-        };
-        let init_header = if self.version >= qt_6_5 {
-            // With Qt6.5 the Q_INIT_RESOURCE macro is in the QtResource header
-            "QtCore/QtResource"
-        } else {
-            "QtCore/QDir"
-        };
-        Initializer {
-            file: Some(output_path),
-            init_call: Some(format!("Q_INIT_RESOURCE({name});")),
-            init_declaration: Some(format!("#include <{init_header}>")),
-        }
+    pub fn qrc(&mut self, input_file: impl AsRef<Path>) -> Initializer {
```

Comment:
need to remove these old usages and push everything through `rcc()`

Reply:
We can do that in a follow-up PR.